### DISCUSSION TITLE
以前论坛里的内容怎么办

### DISCUSSION THREAD

**Discussion by LogicCoder** [kbengine/kbengine#1348] [Q&A] [answered]
以前的论坛里积累的大量的内容，能恢复吗？

**Comment by kbengine** [comment]
后面再想办法恢复

**Comment by Tonyemu** [comment]
https://web.archive.org/web/20201125080227/https://www.comblockengine.com/docs/1.0/get-started/createentity.py/

**Reply by Tonyemu** [reply]
我从这个网站找了写快照。希望能帮助到你

**Comment by kebiao** [comment, marked as answer]
https://github.com/kbengine/kbengine/discussions/1367

**Reply by worldg** [reply]
https://web.archive.org/web/20200807054600/https://www.comblockengine.com/docs/1.0/get-started/createentity/
